azurerm/internal/services/network/resource_arm_loadbalancer_nat_rule.go

```diff
@@ -93,6 +87,24 @@ func resourceArmLoadBalancerNatRule() *schema.Resource {
 				ValidateFunc: validate.NoEmptyStrings,
 			},
 
+			"enable_floating_ip": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+
+			"enable_tcp_reset": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+
+			"idle_timeout_in_minutes": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Computed:     true,
```

Comment:
why is this computed?

Reply:
It's the way lb rule is and allows the API to set the default: 
```
idle_timeout_in_minutes:        "4" => ""
```